notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -551,7 +551,7 @@
     "xtalk_overlaps = {}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """Compute the time-domain DPSS filter matrix."""
</pre><p>Can you write up a bit more about what's being done here, perhaps with a link to your paper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>